tests/some-test-lines.R

```diff
@@ -1,28 +0,0 @@
-library(swephR)
-
-#I need to do these others testi snot found
-#library(roxygen2)
-# roxygenise()
-
-swe_set_ephe_path("C:\\ARCHAEOCOSMO\\ephe")
-swe_set_topo(0,50,10)
-test_that("Existing star position", {
```

Comment:
This test still fails on the CI platforms, since there is no stars data there (yet). However, we can skip this test on these platforms. To do so, please add

    skip_on_travis()
    skip_on_appveyor()

to this test.